sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/ProxyOptions.java

```diff
@@ -40,37 +40,26 @@ public class ProxyOptions implements AutoCloseable {
      */
     public static final String PROXY_AUTHENTICATION_TYPE = "PROXY_AUTHENTICATION_TYPE";
 
-    private static final ConfigurationProperty<ProxyAuthenticationType> AUTH_TYPE_PROPERTY =
-        new ConfigurationPropertyBuilder<>(ConfigurationProperties.AMQP_PROXY_AUTHENTICATION_TYPE, s -> ProxyAuthenticationType.valueOf(s))
-            .shared(true)
-            .logValue(true)
-            .defaultValue(ProxyAuthenticationType.NONE)
-            .build();
-
-    private static final ConfigurationProperty<Proxy.Type> TYPE_PROPERTY = new ConfigurationPropertyBuilder<>(ConfigurationProperties.AMQP_PROXY_TYPE, s -> Proxy.Type.valueOf(s))
-            .shared(true)
-            .logValue(true)
-            .defaultValue(Proxy.Type.HTTP)
-            .build();
-
-    private static final ConfigurationProperty<String> HOST_PROPERTY = ConfigurationPropertyBuilder.ofString(ConfigurationProperties.AMQP_PROXY_HOST)
-        .shared(true)
-        .logValue(true)
-        .build();
-
-    private static final ConfigurationProperty<Integer> PORT_PROPERTY = ConfigurationPropertyBuilder.ofInteger(ConfigurationProperties.AMQP_PROXY_PORT)
-        .shared(true)
-        .required(true)
-        .build();
-
-    private static final ConfigurationProperty<String> USER_PROPERTY = ConfigurationPropertyBuilder.ofString(ConfigurationProperties.AMQP_PROXY_USER)
-        .shared(true)
-        .logValue(true)
-        .build();
-
-    private static final ConfigurationProperty<String> PASSWORD_PROPERTY = ConfigurationPropertyBuilder.ofString(ConfigurationProperties.AMQP_PROXY_PASSWORD)
-        .shared(true)
-        .build();
+    private static final ConfigurationProperty<ProxyAuthenticationType> AUTH_TYPE_PROPERTY
```

Comment:
It's a bit weird that ConfigurationPropertyBuilder isn't always on the same line

Reply:
This is due to the constructor length, Eclipse formatting split based on the first call. The cases where the constructor call is longer than the remaining line space forces it to another line as it's preferring to keep all constructor arguments on as few lines as possible.

The cases where it's on the same line there isn't a first call too long to fit on the line